crates/model/src/order.rs

```diff
@@ -740,6 +740,8 @@ pub struct OrderMetadata {
     pub executed_sell_amount_before_fees: U256,
     #[serde_as(as = "HexOrDecimalU256")]
     pub executed_fee_amount: U256,
+    #[serde_as(as = "HexOrDecimalU256")]
+    pub executed_surplus_fee: U256,
```

Comment:
If this is only used for the UI, can we not add it to the existing executed_fee_amount ?